phpcs.ruleset.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+	<description>Generally-applicable sniffs for WordPress plugins</description>
+
+	<rule ref="WordPress-Core">
+		<exclude name="Squiz.Commenting.FunctionComment.ParamCommentFullStop" />
+		<exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" />
+		<exclude name="Generic.PHP.NoSilencedErrors.Discouraged" />
+		<exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification" />
```

Comment:
`WordPress.CSRF.NonceVerification.NoNonceVerification` is a pretty good rule to leave in place.